### ISSUE TITLE
xDebug version 3 is being installed on the php-fpm during build

### ISSUE THREAD

**Issue by jonpage999** [laradock/laradock#2792]
### Description:
I believe that xDebug v3 (Xdebug v3.0.1) during a default build, the Dockerfile uses `pecl install xdebug;` as the source.

The system otherwise uses xDebug v2 config files. As a result, things do not work as expected!

### Expected Behavior:
Current expectation is that it should install xDebug v2, with the bundled xDebug v2 config files.

Future expectation is that it should install xDebug v3, with appropriate xDebug v3 config files.

### Context information:

**Output of `git rev-parse HEAD`**

```
89c9cfe09941bead05d63ec3c0fe9b9173ce0526
```

### Steps to reproduce the issue:

1. Create a new laradock project
2. Set the PHP_FPM_INSTALL_XDEBUG env variable to TRUE
3. Build a project including php-fpm
4. Check output of `phpinfo();`

### Stacktrace & Additional info:

I don't believe that this affects the workspace, which is installing xDebug differently.

**Comment by jonpage999** [comment]
This appears to be easily fixed by replacing:
`pecl install xdebug;`
with:
`pecl install xdebug-2.9.8;`

**Comment by Jedliu** [comment]
> This appears to be easily fixed by replacing:
> `pecl install xdebug;`
> with:
> `pecl install xdebug-2.9.8;`

**You saved my day!** I've been struggling a few days to make Xdebug running. My local dev environment doesn't work anymore after rebuilding. The web routes work but it can't be debugged while the API routes failed with an error.
```
Nginx 502 Bad Gateway
```

However, this fix doesn't work for **workspace container** as the installation method is different.
```
apt-get install -y php${LARADOCK_PHP_VERSION}-xdebug
```
Any idea to configure this? Thank you!

**Comment by lionslair** [comment]
On the other thread I said turning off xdebug solved it for me and it has. I think it was the latest point release of php 7.4 and the current xdebug.

I did testing the week before on many os's I did probably 12 full builds not an issue. So I'm not sure it's an issue with xdebug or php 7.4 but it was the lastest php 7.4 point release and xdebug that this started occuring. 

I tested every other aspect of laradock imagest old images and had the same issue. The common change was the installation of the latest point release of php 7.4 with xdebug.

I only found the fix buy starting to disable elements one by one. Xdebug being the first. 

I had no log or error anywhere that lead me to it. Just pot luck that xdebug seemed to be the issue. 

Wasn't a fun Monday.
